x-pack/plugins/fleet/common/constants/preconfiguration.ts

```diff
@@ -72,6 +72,8 @@ export const AUTO_UPDATE_PACKAGES = autoUpdatePackages.map((name) => ({
   version: PRECONFIGURATION_LATEST_KEYWORD,
 }));
 
+export const MANAGED_PACKAGES = [...DEFAULT_PACKAGES, ...AUTO_UPDATE_PACKAGES];
```

Comment:
I found the naming kind of confusing we already use the managed term for the preconfigured policy (Probably a bad naming) but what about a more explicit name here like `KEEP_POLICIES_UP_TO_DATE_PACKAGES`